backend/README.md

```diff
@@ -0,0 +1,26 @@
+## Backend scripts to load and transform data
+
+### Requirements:
+Minimum node version: 8.5
+
+#### Installation:
+```
+npm install
+```
+You will need a credentials.json generated from the google developer for sheets api v4 in the base folder
```

Comment:
Can you explain how this works a bit more? What are the steps to take?

Reply:
For CI, I think its simply:
- Set the env variables
- Run `npm install`
- Run `npm start`

Then it should spew out some messages as it goes along